.github/CODEOWNERS

```diff
@@ -34,6 +34,7 @@ Cargo.toml
 /project/ @4e6 @jaroslavtulach @hubertp
 /test/ @jdunkerley @radeusgd @GregoryTravis
 /tools/ @4e6 @jaroslavtulach @radeusgd
+/tools/http-test-helper/ @radeusgd @jdunkerley @GregoryTravis
```

Comment:
I think it makes sense to hand over reviewing the http-test-helper that is solely used for Enso tests to the libraries team, no need to bother engine with it 🙂